### ISSUE TITLE
Add a tool build to release packages

### ISSUE THREAD

**Issue by tomalec** [Starcounter/Starcounter#3243]
It would be great to have a tool to publish app packages.

Even though I have used a grunt tasks I have prepared some time ago, it took me 2hours publish packages for 5 apps, and I'm still not done.

Could we prepare a tool build in star/staradmin or even npm module - just the one simple thing that could be installed globally, not per repo, that would:

1. prepare a package

  1. Build an app if needed,
  1. Update `package.config` with current date,
  1. Update `package.config` with Starcounter version used to build, or at least the one currently installed,
  1. Bump the app version (major/minor/patch/prerelease) in `package.config` (and `bower.json`, `package.json` if applicable)
  2. Do whatever `package.bat` is supoposed to do (it is now a bit inconsistent across apps),

2. Let me upload it to given store with some lightweight parameter or config.

My grunt task covers almost entire step 1, but
 - I do not thing grunt should be a tool used for Satrcounter apps, as we were supposed to collapse the stack,
 - It requires per-repo config (yet another file, that bloat, and requires sync and maintenance across repolepsy),
 - It requires `build.bat` and `package.bat` that does not always are shipped/work/produce same result

**Comment by miyconst** [comment]
+1, this was discussed and required so many times, maybe it's time to finally implement this tool.

**Comment by warpech** [comment]
@andwah how does the above compare to `starpack`?

I believe `starpack` covers all of the above requests apart from `Bump the app version`.

**Comment by andwah** [comment]
> I believe `starpack `covers all of the above requests apart from Bump the app version.

True, it will be a bit tricky but not impossible to also bump the version number with `starpack`. `starpack` uses the already build assembly to create a package so it needs to be rebuilt after version number is bumped.
